tests/component/views/create-dataset/CreateDataset.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import CreateDatasetContainer from '../../../../src/views/create-dataset/CreateDatasetContainer'
+
+describe('Form component', () => {
+  beforeEach(() => {
+    cy.customMount(<CreateDatasetContainer />)
+  })
+  it('can submit a valid form', () => {
+    cy.findByText(/Create Dataset/i).should('exist')
+    cy.findByText(/Title/i).should('exist')
+
+    cy.get('input[name="createDatasetTitle"]')
+      .should('exist')
+      .type('Test Dataset Title')
+      .should('have.attr', 'required', 'required')
+      .and('have.value', 'Test Dataset Title')
```

Comment:
Use testing library syntax here

Reply:
I am using similar specs to https://github.com/IQSS/dataverse-frontend/commit/9146a2cbaa51a19b850e2d3c7246967ffe7a8c3b, should both be changed?